src/Libraries/RevitNodes/Elements/Floor.cs

```diff
@@ -125,7 +125,7 @@ public static Floor ByOutlineTypeAndLevel(Curve[] outlineCurves, FloorType floor
                 throw new ArgumentNullException("outlineCurves");
             }
 
-            var floor = ByOutlineTypeAndLevel(PolyCurve.ByJoinedCurves(outlineCurves), floorType, level);
+            var floor = ByOutlineTypeAndLevel(PolyCurve.ByJoinedCurves(outlineCurves, 0.001, false), floorType, level);
             DocumentManager.Regenerate();
```

Comment:
According to https://github.com/DynamoDS/Dynamo/wiki/API-Changes-in-Dynamo-2.16.0 , PolyCurve.ByJoinedCurves(IEnumerable<Curve> curves, double joinTolerance) has been deprecated and replaced with PolyCurve.ByJoinedCurves(IEnumerable<Curve> curves, double joinTolerance, bool trimCurves, double trimLength). We have to change the code accordingly.